src/sentinel/azext_sentinel/custom.py

```diff
@@ -372,31 +373,33 @@ def _create_or_update_detection(
 
 
 def _link_playbook(
-        security_client: SecurityClient, rule_id: str, playbook_name: str
+    security_client: SecurityClient, rule_id: str, playbook_name: str
 ) -> ActionResponse:
 
-    existing_playbooks: List[
-        ActionResponse
-    ] = security_client.list_actions_by_alert_rule(rule_id=rule_id).value
-    if existing_playbooks and existing_playbooks[0].name == playbook_name:
-        linked_playbook: ActionResponse = existing_playbooks[0]
-    else:
-        _unlink_all_playbooks(security_client=security_client, rule_id=rule_id)
-        playbook = security_client.get_operation(
-            operation_type=OperationType.WORKFLOW, operation_id=playbook_name
-        )
-        trigger_uri = f"{playbook.access_endpoint}{SENTINEL_POST_ALERT_TRIGGER_PATH}"
-        action_request = ActionRequest(
-            logic_app_resource_id=playbook.id, trigger_uri=trigger_uri,
-        )
-        linked_playbook = security_client.create_or_update_operation(
-            operation_type=OperationType.ACTION,
-            operation_id=playbook_name,
-            operation=action_request,
-            rule_id=rule_id,
-        )
+    # TODO: Uncomment the following lines when all the existing playbooks' trigger uris get fixed
```

Comment:
Consider adding a --force-link-playbook option to the CLI that does this so that we can do this from time to time.

If you not, I'd prefer not merging this since company specific changes should not be merged into this public repo.